woocommerce/woocommerce
# [CYS - Full Composability]: When the user is in the "Design your homepage" menu, hovering over the header or footer will should not make them highlighted

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/48252



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you have the last version of Gutenberg enabled.
2. Make sure you have the WooCommerce beta tester plugin installed.
3. Head over to `Tools > WCA Test Helper > Features`.
4. Make sure you see the `pattern-toolkit-full-composability` on the list.
5. Enable it.
6. Head over to WooCommerce > Home > Customize your store.
7. Click on `Design Your Homepage`.
8. Make sure you cannot select the header or the footer by clicking and hovering on them.
9. Go back and click on `Choose your header`.
10. Make sure can select the header by clicking and hovering on it.
11. Go back and click on `Choose your footer`.
10. Make sure can select the footer by clicking and hovering on it.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
CYS - Hovering over the header or footer on the "Design your homepage" section should not make them highlighted.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
